app/experimenter/experiments/models.py

```diff
@@ -1025,13 +1025,19 @@ def latest_change(self):
     def latest_review_request(self):
         return (
             self.all()
-            .filter(NimbusChangeLog.Filters.IS_REVIEW_REQUEST)
+            .filter(
+                NimbusChangeLog.Filters.IS_REVIEW_REQUEST
+                | NimbusChangeLog.Filters.IS_UPDATE_REVIEW_REQUEST
+            )
             .order_by("-changed_on")
```

Comment:
oh we can change this by `-updated_date_time`

Reply:
Looks like there's a few places where we can update `changed_on` -> `updated_date_time`. I'll file a bug for this :D

Reply:
I don't think so?  `-_updated_date_time` is on `NimbusExperiment`, but here we're querying `NimbusChangeLog`?

Reply:
right, I got confused with Nimbus Experiment Manager